src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,46 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
```

Comment:
you can make destructuring from `movie`
```suggestion
export const MovieCard = ({ movie }) => {
  const { ... } = movie;

    return (
```